src/audio/pipeline/pipeline-schedule.c

```diff
@@ -94,6 +94,20 @@ static struct task *pipeline_task_init(struct pipeline *p, uint32_t type,
 	return &task->task;
 }
 
+int pipeline_schedule_config(struct pipeline *p, uint32_t sched_id,
+			     uint32_t core, uint32_t period,
+			     uint32_t period_mips, uint32_t frames_per_sched,
+			     uint32_t time_domain)
+{
+	p->sched_id = sched_id;
+	p->core = core;
+	p->period = period;
+	p->period_mips = period_mips;
+	p->frames_per_sched = frames_per_sched;
+	p->time_domain = time_domain;
```

Comment:
interesting, looks like some of these parameters weren't used at all until now?

Reply:
They were, but the pipeline contained the IPC specific structure and it copied it in bulk.